content/_data/changelogs/weekly.yml

```diff
@@ -19683,6 +19683,22 @@
   # pull: 7791 (PR title: Update dependency webpack to v5.77.0)
   # pull: 7793 (PR title: Bump jenkins-test-harness from 1949.vb_b_37feefe78c to 1952.v3a_b_0cd3f5a_03)
 
+  - version: '2.399'
+    date: 2023-04-04
+    changes:
+      - type: rfe
+        category: rfe
+        authors:
+          - dduportal
+          - markewaite
+          - smerle
+          - lemeurherve
+        references:
+          - url: https://github.com/jenkins-infra/release/pull/358
+            title: pull 358
+        message: |-
+          Sign war file and Windows installer with new code signing certificate.
```

Comment:
War is defined as "a state of usually open and declared armed hostile conflict between states or nations" or "a state of hostility, conflict, or antagonism". A WAR file (**W**eb **A**pplication **R**esource or **W**eb application **AR**chive) is a computer file format. This PR used the former when the latter was meant.

CC @kmartens27

Reply:
Thanks very much @basil, I've compiled all the feedback into [this PR](https://github.com/jenkins-infra/jenkins.io/pull/6274).